### DISCUSSION TITLE
Installing Frigate on MacOS

### DISCUSSION THREAD

**Discussion by DmitrySlav** [blakeblackshear/frigate#3343] [Ask A Question]
Dear friends, I have a MacMini with Intel Core i7 and 8 Gb RAM which I use as a work computer for basic tasks and also runs some applications such as Home Assistant, Plex Server, Torrent client, and Docker with FaceBox. I'd like to install Frigate on my MacMini, but couldn't find any guide. My Linux knowledge and experience are very basic, so I will not be able to do it without any support. I am also afraid to damage my MacOS with the specified applications installed. I hope that the already installed Docker can help me with this task. Please help me find any guide for installing Frigate on my MacOS.

**Comment by NickM-27** [comment]
Frigate is a docker container, the setup guide should be work as is using docker-compose, is there a reason why it seems different?

https://docs.frigate.video/installation

**Reply by DmitrySlav** [reply]
What docker image I should choose?

**Reply by NickM-27** [reply]
probably best to start with `blakeblackshear/frigate:stable-amd64`
